### PR TITLE
Fix energy cost comment discrepancies from #62180

### PR DESCRIPTION
**Pull request by Brambor**
#### Summary
None

#### Purpose of change

In #62180 there were comments in json that disagreed with the values.

#### Describe the solution

Made the values to what the comments say.
Assumption in drying out meat was that 10g of stuff (154 rounded to 150, 155 rounded to 160) per 1u of heat, as supported by the other entries.

#### Testing

None testing done, game is still compiling......

### CODE REVIEW

**Comment by Brambor** [comment]
@anoobindisguise I would love you to review my fixes so I don't introduce more discrepancies.

File: data/json/recipes/food/dry.json [hunk 1]
```diff
@@ -121,7 +121,7 @@
     "time": "20 m",
     "autolearn": true,
     "batch_time_factors": [ 67, 5 ],
-    "tools": [ [ [ "dehydrating_heat", 15, "LIST" ] ], [ [ "surface_heat", 8, "LIST" ] ] ],
+    "tools": [ [ [ "dehydrating_heat", 19, "LIST" ] ], [ [ "surface_heat", 8, "LIST" ] ] ],
     "//": "194g of stuff",
```
Comment (Brambor) [inline data/json/recipes/food/dry.json:125]: Are beans heavy too? Should they have 8u?

File: data/json/recipes/food/dry.json [hunk 2]
```diff
@@ -361,7 +361,7 @@
     "time": "20 m",
     "autolearn": true,
     "batch_time_factors": [ 67, 5 ],
-    "tools": [ [ [ "dehydrating_heat", 21, "LIST" ] ], [ [ "surface_heat", 2, "LIST" ] ] ],
+    "tools": [ [ [ "dehydrating_heat", 21, "LIST" ] ], [ [ "surface_heat", 8, "LIST" ] ] ],
```
Comment (Brambor) [inline data/json/recipes/food/dry.json:364]: Kernels are heavy so 8u.

**Review by Brambor — COMMENTED** [review]
Not 100% sure, please check

**Comment by Maleclypse** [comment]
@anoobindisguise please review?

**Comment by anoobindisguise** [comment]
Hey sorry for the delay this all looks good. Just silly errors on my end, thanks for fixing these.

**Pull request merged**